### PR TITLE
added Octokit.PullRequest.RequestedTeams property

### PR DESCRIPTION
**Pull request by ch1seL**
Related to #2122

### CODE REVIEW

File: Octokit/Models/Response/PullRequest.cs [hunk 1]
```diff
@@ -242,6 +243,11 @@ public bool Merged
         /// Users requested for review
         /// </summary>
         public IReadOnlyList<User> RequestedReviewers { get; protected set; }
+        
+        /// <summary>
+        /// Teams requested for review
+        /// </summary>
+        public IReadOnlyList<Team> RequestedTeams { get; protected set; }
```
Comment (shiftkey) [inline Octokit/Models/Response/PullRequest.cs]: Given we have `RequestedReviewers` above that is configured in the same way, I think this is all we need to do.

Is there any other feedback you're looking for @ch1seL?

Comment (ch1seL) [reply Octokit/Models/Response/PullRequest.cs]: This is half the success of issue #2122
I think we can merge it but the issue will not be resolved totally

Comment (shiftkey) [reply Octokit/Models/Response/PullRequest.cs]: Now I understand. I'll merge this and add some more context to #2122 about the second item on the list once I know what needs to change.

**Comment by shiftkey** [comment]
release_notes: added `RequestedTeams` property to `PullRequest` response model

**Pull request merged**